src/language/c-helpers.rkt

```diff
@@ -0,0 +1,25 @@
+#lang typed/racket/no-check
+(require racket/string "../configuration.rkt")
+(provide (all-defined-out))
+
+(: define-line (Output-Port -> ((Syntaxof Symbol) Syntax -> Void)))
+(define ((define-line in) s v)
+  (define n? (syntax->datum v))
+  (define str (string-replace (symbol->string (syntax-e s)) #rx"-|/" "_"))
+  (displayln
+   (string-append
+    "#define "
+    str
+    " "
+    (if (number? n?)
+        (number->string n?)
+        (string-append "printf(\"I do not know what is " str "\\n\");exit(-1)")))
```

Comment:
Why not just raise a racket error here?

Reply:
Because we allows it. We have this entry:
`(DYN-UNIT-IMDT (bitwise-ior (arithmetic-shift data:UNIT-IMDT data:DYN-IMDT-SHIFT) data:DYN-UNIT-TAG))`.